primitives/src/pallet_traits/frame_system_config.rs

```diff
@@ -58,7 +58,6 @@ pub trait FrameSystemConfig {
 		+ Copy
 		+ sp_std::hash::Hash
 		+ sp_std::str::FromStr
-		+ MaybeMallocSizeOf
```

Comment:
Was removed it seems, check out: https://github.com/paritytech/substrate/blob/master/frame/system/src/lib.rs#L252-L253